src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
+using System.Linq;
 using Microsoft.ML.Data;
```

Comment:
Is Linq required?

Reply:
Not strictly; the new code uses .FirstOrDefault(), but that can of course be avoided if preferred

---
In reply to: [445187333](https://github.com/dotnet/machinelearning/pull/5236#discussion_r445187333) [](ancestors = 445187333)